x-pack/legacy/plugins/apm/public/components/shared/charts/Legend/index.js

```diff
@@ -47,7 +48,11 @@ export default class Legend extends PureComponent {
         fontSize={fontSize}
         {...rest}
       >
-        <Indicator color={color} radius={radius} />
+        {indicator ? (
+          indicator({ color, radius })
```

Comment:
It looks like the `indicator` function doesn't take any args?

Reply:
I figured I would pass the same props to `indicator` as the component gets, but it might be overkill because indeed I'm not using it now. Remove?

Reply:
I think if unused just remove

Reply:
removed 👍